src/tracing/server.coffee

```diff
@@ -0,0 +1,25 @@
+###
+This server response to ui on a /vizc end point and returns queried JSON as reponse
+###
+express = require 'express'
+bodyparser = require 'body-parser'
+varz = require 'express-varz'
+{conf} = require 'rainier/conf'
+port = conf.get 'vizc_server_port'
+vizJsonCreator = require './vizceral_json_file_creator_nodes'
+
+server = express()
+server.use bodyparser.json limit: '10mb'
```

Comment:
The size of 140 records took 4kb space. We plan to query for last 5 minutes. 1mb = 35k records. Good  enough even in production right?